### PR TITLE
data: anekemichael

### PR DESCRIPTION
**Pull request by anekemichael**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5838"></a>

### CODE REVIEW

**Comment by anekemichael** [comment]
Please what is the problem with my file.?

File: data/anekemichael.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    {
+      "name": "Follow Me on Github",
+      "url": "https://github.com/anekemichael/",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Subscribe to My Channel on Youtube",
+      "url": "https://youtube.com/@geekybuds/",
+      "icon": "FaYoutube"
+    },
```
Comment (khushalarora) [inline data/anekemichael.json:30]: Please remove this extra **' Comma '** from this line :+1: 
```Suggestion
    }
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hi @anekemichael, thank you for contributing   :tada:

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by anekemichael** [comment]
Thanks so much @SaraJaoude

**Review by anekemichael — COMMENTED** [review]
Please i would like to add some links in my commit

**Comment by SaraJaoude** [comment]
> Please i would like to add some links in my commit

If you would like to add links to your Profile, you will need to open a new Pull Request.  I suggest you look at the Editing Guide in Docs and follow the steps: https://linkfree.eddiehub.io/docs/how-to-guides/editing

**Comment by anekemichael** [comment]
Alright.. Thanks so much

**Pull request merged**